### PR TITLE
[WIP] Dockerflow

### PR DESCRIPTION
**Pull request by rail**
* `scripts/push-dockerimage.sh` is almost a verbatim copy from balrog
* `.neutrinorc.js` changes to address the routes and disable https if needed (in Dockerflow it's handled by the proxy)
* `secrets:get:repo:github.com/mozilla-releng/shipit-v2:dockerhub` is accessable by the following roles:
  * `repo:github.com/mozilla-releng/shipit-v2:branch:master`
  * `repo:github.com/mozilla-releng/shipit-v2:release`

### CODE REVIEW

**Comment by rail** [comment]
https://tools.taskcluster.net/groups/Zz3C4QL-QHCbnL6j_DveWw is green!

File: scripts/push-dockerimage.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    echo "Dockerhub password not set, can't continue!"
+    exit 1
+fi
+if [ ${#tags[@]} -eq 0 ]; then
+    echo "Must pass at least one tag"
+    exit 2
+fi
+
+commit=$(git rev-parse HEAD)
+version=$(grep '"version": ' package.json | awk -F'"' '{print $4}')
```
Comment (MihaiTabara) [inline scripts/push-dockerimage.sh]: This seems fragile should we ever accidentally change that `package.json` structure or even add whitespace or stuff. But I suppose it works for now

Comment (rail) [reply scripts/push-dockerimage.sh]: Yeah, I just didn't want to include jq in the image... Probably I should use python instead.

File: scripts/push-dockerimage.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    echo "Dockerhub password not set, can't continue!"
+    exit 1
+fi
+if [ ${#tags[@]} -eq 0 ]; then
+    echo "Must pass at least one tag"
+    exit 2
+fi
+
+commit=$(git rev-parse HEAD)
+version=$(cat package.json| python -c 'import json, sys; a = json.load(sys.stdin); print a["version"];')
```
Comment (MihaiTabara) [inline scripts/push-dockerimage.sh]: 👍

**Pull request merged**